website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -448,6 +450,290 @@ $ consul-k8s proxy read backend-658b679b45-d5xlb -o raw
 }
 ```
 
+### `proxy log`
+
+The `proxy log` command allows you to inspect and modify the logging configuration of Envoy proxies running on a given Pod.
+
+```shell-session
+$ consul-k8s proxy log <PODNAME> <OPTIONS>
+```
+
+The command takes a required value, `<PODNAME>`. This should be the full name
+of a Kubernetes Pod. If a Pod is running more than one Envoy proxy managed by
+Consul, as in the [Multiport configuration](/consul/docs/k8s/connect#kubernetes-pods-with-multiple-ports),
+the terminal displays configuration information for all proxies in the pod.
+
+The following options are available.
+
+| Flag                               | Description                                                                                                                                                                                                                                                                                                              | Default                                                                                                                |
+| ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------- |
+| <nobr>`-namespace`, `-n`</nobr>    | `String` Specifies the namespace containing the target Pod.                                                                                                                                                                                                                                                              | Current [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) namespace. |
+| <nobr>`-update-level`, `-u`</nobr> | `String`  Specifies the logger (optional) and the level to update. <p>Use the following format to configure the same level for loggers: `-update-level <LEVEL>`.</p> <p>You can also specify a comma-delineated list to configure levels for specific loggers, for example: `-update-level grpc:warning,http:info`.</p>  | none                                                                                                                   |
```

Comment:
@trujillo-adam what do you think of the text here? I modified it slightly from your suggestion to show that they can either blanket update all the level for all the loggers or target specific ones